README.md

```diff
@@ -1 +1,274 @@
+      <div id="after" class="map"></div>
+    </div>
+    <script>
+      var beforeMap = new maplibregl.Map({
+        container: "before",
+        style:
+          "https://api.maptiler.com/maps/hybrid/style.json?key=get_your_own_OpIi9ZULNHzrESv6T2vL",
+        center: [7.221275, 50.326111],
+        zoom: 15,
+      });
+
+      var afterMap = new maplibregl.Map({
+        container: "after",
+        style:
+          "https://api.maptiler.com/maps/streets/style.json?key=get_your_own_OpIi9ZULNHzrESv6T2vL",
+        center: [7.221275, 50.326111],
+        zoom: 15,
+      });
+
+      // A selector or reference to HTML element
+      var container = "#comparison-container";
+
+      var map = new maplibregl.Compare(beforeMap, afterMap, container, {
+        // Set this to enable comparing two maps by mouse movement:
+        // m ousemove: true
+      });
+    </script>
+  </body>
+</html>
+```
+
+### Usage
+
+```js
+var beforeMap = new maplibregl.Map({
+container: "before",
+style: "https://demotiles.maplibre.org/style.json",
+center: [7.221275, 50.326111],
+zoom: 5,
+});
+
+var afterMap = new maplibregl.Map({
+container: "after",
+style:
+    "https://vectortiles.geo.admin.ch/styles/ch.swisstopo.leichte-basiskarte.vt/style.json",
+center: [7.221275, 50.326111],
+zoom: 5,
+});
+
+// A selector or reference to HTML element
+var container = '#comparison-container';
+
+var map = new maplibregl.Compare(beforeMap, afterMap, container, {
+  mousemove: true, // Optional. Set to true to enable swiping during cursor movement.
+  orientation: 'vertical' // Optional. Sets the orientation of swiper to horizontal or vertical, defaults to vertical
+});
+```
+
+### Methods
+
+```js
+compare = new mapboxgl.Compare(beforeMap, afterMap, container, {
+  mousemove: true, // Optional. Set to true to enable swiping during cursor movement.
+  orientation: 'vertical' // Optional. Sets the orientation of swiper to horizontal or vertical, defaults to vertical
+});
+
+// Get Current position - this will return the slider's current position, in pixels
+compare.currentPosition;
+
+// Set Position - this will set the slider at the specified (x) number of pixels from the left-edge or top-edge of viewport based on swiper orientation
+compare.setSlider(x);
+
+//Listen to slider movement - and return current position on each slideend
+compare.on('slideend', (e) => {
+  console.log(e.currentPosition);
+});
+
+//Remove - this will remove the compare control from the DOM and stop synchronizing the two maps.
+compare.remove();
+```
+
+## Live examples
+
+[Compare hybrid with streets](https://rawcdn.githack.com/astridx/astridx.github.io/a9d7297a4fe1e3a4d7ebeb1e4e662fd1339ef3b5/maplibreexamples/plugins/maplibre-gl-compare-swipe-between-maps.html)
+
+[Compare swisstopo with demotiles](https://rawcdn.githack.com/astridx/astridx.github.io/a9d7297a4fe1e3a4d7ebeb1e4e662fd1339ef3b5/maplibreexamples/plugins/maplibre-gl-compare-swipe-between-maps_.html)
```

Comment:
Are the links OK? I use files in my repo.

Reply:
Yes, thanks for the examples.